packages/playground/wordpress/src/index.ts

```diff
@@ -131,27 +131,34 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 			if (!$user) {
 				return;
 			}
+			/**
+			 * This approach is described in a comment on
+			 * https://developer.wordpress.org/reference/functions/wp_set_current_user/
+			 */
 			wp_set_current_user( $user->ID, $user->user_login );
 			wp_set_auth_cookie( $user->ID );
 			do_action( 'wp_login', $user->user_login, $user );
 			setcookie('playground_auto_login_already_happened', '1');
-		}
 
+			/**
+			 * Reload page to ensure the user is logged in correctly.
+			 * WordPress uses cookies to determine if the user is logged in,
+			 * so we need to reload the page to ensure the cookies are set.
+			 *
+			 * The redirect should only run if the current PHP request is
+			 * a HTTP request. If it's a PHP run, there is nothing to reload.
+			 */
+			if (!empty($_SERVER['REQUEST_URI'])) {
```

Comment:
@adamziel I would love to get your thoughts on this.

If the current script execution is a PHP run and not an HTTP request, there is no way for us to redirect.
But, there is also no way for us to set the cookies correctly. Should we just skip autologin during PHP runs? 
Is there a better way of detecting this then checking if the `$_SERVER` global is populated?